libs/blocks/caas-config/caas-config.css

```diff
@@ -97,6 +97,50 @@ dd.content .sort-options > .field {
   margin-bottom: 20px;
 }
 
-.config-panel::-webkit-scrollbar { 
+.config-panel::-webkit-scrollbar {
   display: none;  /* Safari and Chrome */
 }
+
+/* Extends multifield */
+[class^="multifield filters"] .multifield-header h3 {
+  display: flex; justify-content: space-between;
```

Comment:
I know this is not customer-facing, but we should try and keep a similar formatting to the rest of the project and have one rule per line

Reply:
Good catch, thanks!